DDG4/python/DDSim/DD4hepSimulation.py

```diff
@@ -346,6 +346,12 @@ def run(self):
       lcOut.EventParametersString, lcOut.EventParametersInt, lcOut.EventParametersFloat = eventPars
       lcOut.RunNumberOffset = self.meta.runNumberOffset if self.meta.runNumberOffset > 0 else 0
       lcOut.EventNumberOffset = self.meta.eventNumberOffset if self.meta.eventNumberOffset > 0 else 0
+    elif self.outputFile.endswith("edm4hep.root"):
+      e4Out = simple.setupEDM4hepOutput('EDM4hepOutput', self.outputFile)
+#      e4Out.RunHeader = self.meta.addParametersToRunHeader(self)
```

Comment:
why this comment line?

Reply:
This is still WIP. There are many lines commented out that deal with event and run headers and meta data parameters as these still don't exist in EDM4hep...